### PR TITLE
fix(QueryEditor): better report timeout errors

### PR DESCRIPTION
**Pull request by ariser**

### CODE REVIEW

File: src/containers/Tenant/QueryEditor/QueryResult/QueryResult.js [hunk 1]
```diff
@@ -94,31 +94,33 @@ function QueryResult(props) {
...
-                            <ResultIssues data={error} />
+                            <ResultIssues data={error.data} />
                         </div>
                     </Fullscreen>
                 )}
             </React.Fragment>
         ) : (
-            <span>{error?.data ?? error}</span>
+            <div className={b('error')}>
+                {error?.message || error?.data || error}
```
Comment (artemmufazalov) [inline src/containers/Tenant/QueryEditor/QueryResult/QueryResult.js:114]: Can error data be an object? Or error itself? Won't it fail with "Objects are not valid as a React child"? Maybe it's better to use `prepareQueryError` as it seems to fit here. Or at least parse it to string

File: src/services/api.js [hunk 2]
```diff
@@ -185,7 +185,10 @@ export class YdbEmbeddedAPI extends AxiosWrapper {
                 timeout: 600000,
```
Comment (artemmufazalov) [inline src/services/api.js:185]: Is it a backend timeout? Why 2 timeouts?

File: src/components/QueryExecutionStatus/QueryExecutionStatus.tsx [hunk 3]
```diff
@@ -1,29 +1,46 @@
...
+import questionIcon from '../../assets/icons/question.svg';
+
 import './QueryExecutionStatus.scss';
 
 const b = cn('kv-query-execution-status');
 
 interface QueryExecutionStatusProps {
     className?: string;
-    hasError?: boolean;
+    error?: Record<string, any>;
```
Comment (artemmufazalov) [inline src/components/QueryExecutionStatus/QueryExecutionStatus.tsx:16]: I think It is better to introduce some interface `IQueryResponseError` and at least specify  properties that could be present. It makes it much easier to understand

File: src/services/api.js (hunk 2 above)
Comment (ariser) [reply src/services/api.js:185]: one for backend operations, one for the client request

File: src/containers/Tenant/QueryEditor/QueryResult/QueryResult.js [hunk 4]
```diff
@@ -94,31 +100,33 @@ function QueryResult(props) {
...
-                            <ResultIssues data={error} />
+                            <ResultIssues data={error.data} />
                         </div>
                     </Fullscreen>
                 )}
             </React.Fragment>
         ) : (
-            <span>{error?.data ?? error}</span>
+            <div className={b('error')}>
+                {prepareQueryError(error)}
```
Comment (artemmufazalov) [inline src/containers/Tenant/QueryEditor/QueryResult/QueryResult.js:120]: It seems error could be undefined here. And `prepareQueryError` expects error to be defined. 

https://github.com/ydb-platform/ydb-embedded-ui/blob/main/src/utils/query.ts#L188

Comment (ariser) [reply src/containers/Tenant/QueryEditor/QueryResult/QueryResult.js]: fixed

**Review by artemmufazalov — DISMISSED** [review]

**Review by artemmufazalov — APPROVED** [review]

**Pull request merged**